### ISSUE TITLE
stdout_syslog depends on stdout_logfile

### ISSUE THREAD

**Issue by scottkof** [Supervisor/supervisor#1375]
**Expected behavior:** `stdout_syslog` and `stdout_logfile` are independent.
Given the following configs
```
  stdout_logfile = NONE
  stdout_syslog = true
```
child process logs should be logged to syslog.
<hr>

**Actual behavior:** `stdout_syslog` depends on `stdout_logfile`.
Given the following configs
```
  stdout_logfile = NONE
  stdout_syslog = true
```
child process logs are _not_ logged to syslog.

However, given the following configs
```
  stdout_logfile = AUTO
  stdout_syslog = true
```
child process logs are written to syslog (and logfile).

**Comment by scottkof** [comment]
[PR #1376](https://github.com/Supervisor/supervisor/pull/1376) is a fix for this issue.

**Comment by mnaberez** [comment]
Fixed in #1376 (committed as ab1e07cc6d502c452f00c1e2d88989e91e75f294).
